bodhi/server/models.py

```diff
@@ -3541,6 +3541,33 @@ def __json__(self, request=None, anonymize=False):
 
         return result
 
+    @staticmethod
+    def comment_on_test_gating_status_change(target, value, old, initiator):
+        """
+        Place comment on the update when ``test_gating_status`` changes.
+
+        Args:
+            target (Update): The compose that has had a change to its state attribute.
+            value (EnumSymbol): The new value of the test_gating_status.
+            old (EnumSymbol): The old value of the test_gating_status
+            initiator (sqlalchemy.orm.attributes.Event): The event object that is initiating this
+                transition.
+        """
+        if value != old:
+            target.comment(
+                Session(),
+                u"This update test_gating_status has been changed to '%s'." % (value),
```

Comment:
Just a suggestion, but we can use the cool new Python 3 f-strings feature now!

```suggestion
                f"This update test_gating_status has been changed to '{value}'.",
```

Also, since this is meant to be human readable, maybe we want to say "test gating status" or something similar (i.e., no _'s).

Reply:
Good hint! I forgot that Bodhi is now py3 only.